<ui version="4.0" >
 <class>TaskWindow</class>
 <widget class="QMainWindow" name="TaskWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TaskTimer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTableView" name="taskTable" >
      <property name="contextMenuPolicy" >
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>View</string>
    </property>
    <addaction name="actionHideDone" />
   </widget>
   <widget class="QMenu" name="menuTask" >
    <property name="title" >
     <string>Task</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionStart" />
    <addaction name="actionStopAll" />
    <addaction name="actionRemove" />
    <addaction name="actionDone" />
    <addaction name="actionNewSession" />
    <addaction name="separator" />
    <addaction name="actionPrio1" />
    <addaction name="actionPrio2" />
    <addaction name="actionPrio3" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTask" />
   <addaction name="menuView" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>TaskTimer</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionStart" />
   <addaction name="actionStopAll" />
   <addaction name="actionRemove" />
   <addaction name="actionDone" />
   <addaction name="actionNewSession" />
   <addaction name="actionShowNotes" />
  </widget>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
   </property>
   <property name="text" >
    <string>Start</string>
   </property>
   <property name="shortcut" >
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/media-playback-stop.png</normaloff>:/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRemove" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/edit-delete.png</normaloff>:/icons/edit-delete.png</iconset>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDone" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/stock_form-checkbox.png</normaloff>:/icons/stock_form-checkbox.png</iconset>
   </property>
   <property name="text" >
    <string>Done</string>
   </property>
  </action>
  <action name="actionHideDone" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Hide done</string>
   </property>
   <property name="shortcut" >
    <string>T</string>
   </property>
  </action>
  <action name="actionPrio1" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>normal</string>
   </property>
  </action>
  <action name="actionPrio2" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>high</string>
   </property>
  </action>
  <action name="actionPrio3" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>highest</string>
   </property>
  </action>
  <action name="actionStopAll" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/media-playback-stop.png</normaloff>:/icons/media-playback-stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop all</string>
   </property>
   <property name="shortcut" >
    <string>S</string>
   </property>
  </action>
  <action name="actionNewSession" >
   <property name="icon" >
    <iconset resource="../pics/pics.qrc" >
     <normaloff>:/icons/stock_compile.png</normaloff>:/icons/stock_compile.png</iconset>
   </property>
   <property name="text" >
    <string>New Session</string>
   </property>
  </action>
  <action name="actionShowNotes" >
   <property name="text" >
    <string>Show Notes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../pics/pics.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>TaskWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>400</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
